packages/cli/src/commands/sandbox/sandbox_command.test.ts

```diff
@@ -205,6 +205,46 @@ void describe('sandbox command', () => {
     assert.equal(sandboxDeleteMock.mock.callCount(), 1);
   });
 
+  void it('asks to delete the sandbox environment when users send ctrl-C and say yes to delete with profile', async (contextual) => {
+    // Mock process and extract the sigint handler after calling the sandbox command
+    const processSignal = contextual.mock.method(process, 'on', () => {
+      /* no op */
+    });
+    const sandboxStartMock = contextual.mock.method(
+      sandbox,
+      'start',
+      async () => Promise.resolve()
+    );
+
+    const sandboxDeleteMock = contextual.mock.method(sandbox, 'delete', () =>
+      Promise.resolve()
+    );
+
+    // User said yes to delete
+    contextual.mock.method(AmplifyPrompter, 'yesOrNo', () =>
+      Promise.resolve(true)
+    );
+
+    const profile = 'test_profile';
+    await commandRunner.runCommand(`sandbox --profile ${profile}`);
+
+    // Similar to the later 0ms timeout. Without this tests in github action are failing
+    // but working locally
```

Comment:
is it possible something is missing in the code? I don't see this required in line [360](https://github.com/aws-amplify/amplify-backend/pull/1900/files#diff-fe3062a0618a08be0e78b23acc94e8621cd2d865ad1b63da87b19278b8fe06c7R360)

Reply:
possibly a CI flag in the yargs tool?

Reply:
The other test you mentioned is not testing SIGINT.

I took this from test above. I.e. line 197.

I wanted to cover a change where profile is passed to sandbox delete that happens when you say "yes" after CTRL+C.

Reply:
do we need to re-import yargs in our beforeEach? https://github.com/yargs/yargs/blob/c36c571e4e15dfe26be1d919e4991fb6ab6ed9fd/test/yargs.js#L20

Reply:
Afaik we can't re-import yargs while using ESM imports. 

This solution was not invented in this PR.
See https://github.com/aws-amplify/amplify-backend/pull/100#discussion_r1280702279